nbformat/tests/test_sign.py

```diff
@@ -4,9 +4,11 @@
 # Distributed under the terms of the Modified BSD License.
 
 import copy
+import os
 import shutil
 import time
 import tempfile
+import testpath
```

Comment:
Sounds good, the other way felt distinctly wrong but I wondered since these are so often grouped together if it were somehow right.  Fixing things up now and hoping tests pass!